packages/clay-nav/docs/index.js

```diff
@@ -123,4 +124,94 @@ const VerticalNavigation = () => {
 	);
 };
 
-export {Navigation, VerticalNavigation};
+const verticalNavigationCustomItemImportsCode = `import {VerticalNav} from '@clayui/nav';
+import ClayIcon from '@clayui/icon';
+`;
+
+const VerticalNavigationCustomItemCode = `const Component = () => {
+	const noIcons = [1, 5];
+
+	return (
+		<ClayVerticalNav
+			active={6}
+			defaultExpandedKeys={new Set([5])}
+			items={[
+				{
+					id: 1,
+					items: [
+						{
+							id: 2,
+							href: '#nested1',
+							label: 'Nested1',
+						},
+					],
+					label: 'Home',
+				},
+				{
+					id: 3,
+					href: '#2',
+					label: 'About',
+				},
+				{
+					id: 4,
+					href: '#3',
+					label: 'Contact',
+				},
+				{
+					id: 5,
+					items: [
+						{
+							id: 6,
+							href: '#5',
+							label: 'Five',
+						},
+						{
+							id: 7,
+							href: '#6',
+							label: 'Six',
+						},
+					],
+					label: 'Projects',
+				},
+				{
+					id: 8,
+					href: '#7',
+					label: 'Seven',
+				},
+			]}
+			large={false}
+			spritemap={spritemap}
+		>
+			{(item) => (
+				<ClayVerticalNav.Item
+					href={item.href}
+					items={item.items}
+					key={item.id}
+				>
+					{item.label}
+					{noIcons.indexOf(item.id) < 0 && (
```

Comment:
I think this can be made more legible with the use of `includes`:

```suggestion
					{!noIcons.includes(item.id) && (
```